packages/medusa-js/src/resources/customers.ts

```diff
@@ -83,7 +83,7 @@ class CustomerResource extends BaseResource {
    * @return {ResponsePromise<StoreCustomersRes>}
    */
   resetPassword(
-    payload: StorePostCustomersCustomerPasswordTokenReq,
+    payload: StorePostCustomersResetPasswordReq,
```

Comment:
todo: Add import similar to how other types are imported

Reply:
done:)